src/context/viewer-context.tsx

```diff
@@ -37,11 +37,11 @@ function useAuthedViewer() {
   const [viewer, setViewer] = React.useState()
   const viewerId = get(viewer, 'id', null)
   const [loading, setLoading] = React.useState(true)
+  const [loggingOut, setLoggingOut] = React.useState(false)
   const previousViewer = React.useRef(viewer)
-  const authToken = getAccessTokenFromCookie()
 
   useTokenSigner()
-  useAffiliateAssigner(viewerId, authToken)
+  useAffiliateAssigner(viewerId, getAccessTokenFromCookie())
```

Comment:
really the auth token lookup could be service locator style inside of the hook instead of passed in, but I wanted to make the auth token a non-dependency for the useEffect below